src/main/java/com/joyent/manta/cosbench/MantaStorage.java

```diff
@@ -21,6 +21,7 @@
 import com.joyent.manta.exception.MantaErrorCode;
 import com.joyent.manta.http.MantaHttpHeaders;
 import org.apache.commons.io.input.BoundedInputStream;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
If the StringUtils dependency is only used for `StringUtils.EMPTY`, I think it would be best to omit the dependency and just use `""`.

Reply:
makes sense!